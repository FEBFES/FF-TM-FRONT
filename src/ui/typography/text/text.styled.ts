import styled from 'styled-components';

export const SText = styled.p`
  font-weight: 400;
  font-size: 14px;
  color: #575757;
`;
