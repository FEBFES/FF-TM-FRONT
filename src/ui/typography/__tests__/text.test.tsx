import React from 'react';
import { Text } from '../text/text';
import renderer from 'react-test-renderer';

it('SNAPSHOTS: UI - Text component - render', () => {
  const elem = renderer.create(<Text>linka</Text>).toJSON();
  expect(elem).toMatchSnapshot();
});
