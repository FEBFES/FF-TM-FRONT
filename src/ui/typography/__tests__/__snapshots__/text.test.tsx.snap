// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNAPSHOTS: UI - Text component - render 1`] = `
<p
  className="sc-bcPKhP dKWSvg"
>
  linka
</p>
`;
